### PR TITLE
Android 15 with virtio integration

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

File: prod-devel-rcar-virtio.yaml [hunk 1]
```diff
@@ -336,7 +336,7 @@ components:
     sources:
       - type: repo
         url: https://github.com/xen-troops/android_manifest.git
-        rev: "493c9a919949262095c598cbb6c5cd0258dfc58f" # android-14-xenvm-trout-main
+        rev: "android-15-xenvm-trout-main"
```
Comment (rshym) [inline prod-devel-rcar-virtio.yaml:339]: First, this branch doesn't exist on the repo right now.
And even when it is created, this doesn't look good, considering the commit message.
If moving to Android 15 is still a work in progress, it should be done on a separate branch, and the commit should be changed when the upgrade is considered complete.

**Comment by rshym** [comment]
As result of the off-site discussion
`Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

File: prod-devel-rcar-virtio.yaml (hunk 1 above)
Comment (arminn) [reply prod-devel-rcar-virtio.yaml]: Done

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by arminn** [comment]
Added ACKs.

**Pull request merged**